src/main/app/src/app/zaken/intake-afronden-dialog/intake-afronden-dialog.component.html

```diff
@@ -20,12 +30,12 @@
       >
     </mat-radio-group>
   </div>
-  <div>
-    <mat-form-field
-      appearance="fill"
-      class="full-width"
-      *ngIf="formGroup.get('ontvankelijk').value === false"
-    >
+  <div *ngIf="formGroup.get('ontvankelijk').value === false">
```

Comment:
`*ngIf="!formGroup.get('ontvankelijk').value"` ? or do we really need to check it is `false`

Reply:
but then `!formGroup.get('ontvankelijk').value` would suffice